features/step_definitions/find_car_steps.rb

```diff
@@ -0,0 +1,202 @@
+Given /^search page is opened$/ do
+  FindCarPage.open
+end
+
+When /^user fill in Postcode field with (\S+)$/ do |data|
+  FindCarPage.given.enter_data(data)
+end
+
+When /^click the (.+) button$/ do |button_name|
+  FindCarPage.given.click_button(button_name)
+end
+
+Then /^on the page with found cars an? (.+) filter is selected as (.+)$/ do |param1, param2|
```

Comment:
Try to avoid generic names, such as `param`, use more specific words, like `btn_name` and `btn_text`